docs/product/integrations/source-code-mgmt/github/index.mdx

```diff
@@ -36,7 +36,14 @@ Sentry owner, manager, or admin permissions, and GitHub owner permissions are re
 
 The GitHub integration is available for all projects under your Sentry organization. You can connect multiple GitHub organizations to one Sentry organization, but you **cannot** connect a single GitHub organization to multiple Sentry organizations.
 
-## GitHub Enterprise
+## GitHub Enterprise (self-hosted/on-prem)
```

Comment:
If you update the header text, you'll need to update the link in line 11 so that it goes to the right anchor

Reply:
Oh didn't even think about that (and how that might break the existing links for GHE already out there...! I'll undo that change and leave the header as-is. Good feedback 👍